<sect1 id="zend.exception.using">

    <title>Utilizzo delle eccezioni</title>

    <para>
        Tutte le eccezioni lanciate dalle classi del Framework Zend dovrebbero estendere la classe base Zend_Exception.
    </para>

    <example id="zend.exception.using.example">
        <title>Gestione di un'eccezione</title>
        <para>
            Il codice seguente dimostra come gestire un'eccezione lanciata all'interno di una classe del Framework Zend:
        </para>
        <programlisting role="php"><![CDATA[
try {
    // La chiamata a Zend_Loader::loadClass() con una classe non esistente
    // causa il lancio di un'eccezione in Zend_Loader
    Zend_Loader::loadClass('classenonesistente');
} catch (Zend_Exception $e) {
    echo "Catturata eccezione: " . get_class($e) . "\n";
    echo "Messaggio: " . $e->getMessage() . "\n";
    // Altro codice necessario per gestire l'errore
}
]]>
        </programlisting>
    </example>

    <para>
        La documentazione di ogni componente e classe del Framework Zend
        contiene informazioni specifiche su quali metodi lanciano eccezioni,
        le circostanze che causano l'eccezione, e le classi per tutte
        le eccezioni che possono essere lanciate.
    </para>

</sect1>
<!--
vim:se ts=4 sw=4 et:
-->
